src/music/music.repository.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from "@nestjs/common";
+import { CreateMusicDto } from "./dto/create-music.dto";
+import { UpdateMusicDto } from "./dto/update-music.dto";
+import { InjectRepository } from "@nestjs/typeorm";
+import { Music } from "./entities/music.entity";
+import { Repository } from "typeorm";
+
+@Injectable()
+export class MusicRepository {
+    constructor(
+        @InjectRepository(Music)
+        private musicRepoitory: Repository<Music>
+    ) {}
+  create(createMusicDto: CreateMusicDto) {
+    const newMusic = new Music();
```

Comment:
you dont need this you can use just musicRepository.create(createMusicDto) and then save